pyqtgraph/graphicsItems/InfiniteLine.py

```diff
@@ -240,3 +336,50 @@ def setMouseHover(self, hover):
         else:
             self.currentPen = self.pen
         self.update()
+
+    def viewTransformChanged(self):
+        """
+        Called whenever the transformation matrix of the view has changed.
+        (eg, the view range has changed or the view was resized)
+        """
+        self._invalidateCache()
+
+        if self.getViewBox() is not None and isinstance(self.getViewBox(), ViewBox) and self.textItem is not None:
+            self.updateTextPosition()
+
+    def showLabel(self, state):
+        """
+        Display or not the label indicating the location of the line in data
+        coordinates.
+
+        ==============   ==============================================
```

Comment:
sphinx will complain--the table lines need to be at least as wide as the contents of the table.